### PR TITLE
en,zh: release note for 1.4.5

### PR DESCRIPTION
**Pull request by csuzhangxc**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-tidb-operator) that's required for repo owners to accept my contribution.

### Which TiDB Operator version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v1.5 (TiDB Operator 1.5 versions)
- [x] v1.4 (TiDB Operator 1.4 versions)
- [ ] v1.3 (TiDB Operator 1.3 versions)
- [ ] v1.2 (TiDB Operator 1.2 versions)

### What is the related PR or file link(s)?

- This PR is translated from: 
- Other reference link(s):

### CODE REVIEW

File: en/releases/release-1.4.5.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+TiDB Operator version: 1.4.5
+
+## Improvements
+
+- Add the metrics for fine-grained TidbCluster reconcile errors ([#4952](https://github.com/pingcap/tidb-operator/pull/4952), [@coderplay](https://github.com/coderplay))
+- Add the metrics for the reconciler and worker queue to improve observability ([#4882](https://github.com/pingcap/tidb-operator/pull/4882), [@hanlins](https://github.com/hanlins))
+- Introduce `startUpScriptVersion` field for DM master to specify startup script version ([#4971](https://github.com/pingcap/tidb-operator/pull/4971), [@hanlins](https://github.com/hanlins))
+
+## Bug fixes
```
Comment (Ehco1996) [inline en/releases/release-1.4.5.md:17]:
```suggestion
## Bug fixes

- Supress gc when scheduled backup is newly created ([#4940](https://github.com/pingcap/tidb-operator/pull/4940), [@oliviachenairbnb](https://github.com/oliviachenairbnb))
- Make LogBackupTemplate optional in backup CR ([#4956](https://github.com/pingcap/tidb-operator/pull/4956), [@Ehco1996](https://github.com/Ehco1996))
```

File: zh/releases/release-1.4.5.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+TiDB Operator 版本：1.4.5
+
+## 优化提升
+
+- 为 TidbCluster 增加细粒度的调协处理错误相关的 metrics ([#4952](https://github.com/pingcap/tidb-operator/pull/4952), [@coderplay](https://github.com/coderplay))
+- 增加调协处理与 worker 队列相关的 metrics 以提升可观测性 ([#4882](https://github.com/pingcap/tidb-operator/pull/4882), [@hanlins](https://github.com/hanlins))
+- 为 DM master 组件增加 `startUpScriptVersion` 字段以用于指定启动脚本的版本 ([#4971](https://github.com/pingcap/tidb-operator/pull/4971), [@hanlins](https://github.com/hanlins))
+
+## Bug 修复
```
Comment (Ehco1996) [inline zh/releases/release-1.4.5.md:17]:
```suggestion
## Bug 修复

- 在新创建的计划备份中取消 gc ([#4940](https://github.com/pingcap/tidb-operator/pull/4940), [@oliviachenairbnb](https://github.com/oliviachenairbnb))
- 让 Backup CR 字段里的 LogBackupTemplate 字段变成可选值 ([#4956](https://github.com/pingcap/tidb-operator/pull/4956), [@Ehco1996](https://github.com/Ehco1996))
```

File: zh/releases/release-1.4.5.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+发布日期: 2023 年 6 月 TODO 日
+
+TiDB Operator 版本：1.4.5
+
+## 优化提升
+
+- 为 TidbCluster 增加细粒度的调协处理错误相关的 metrics ([#4952](https://github.com/pingcap/tidb-operator/pull/4952), [@coderplay](https://github.com/coderplay))
+- 增加调协处理与 worker 队列相关的 metrics 以提升可观测性 ([#4882](https://github.com/pingcap/tidb-operator/pull/4882), [@hanlins](https://github.com/hanlins))
+- 为 DM master 组件增加 `startUpScriptVersion` 字段以用于指定启动脚本的版本 ([#4971](https://github.com/pingcap/tidb-operator/pull/4971), [@hanlins](https://github.com/hanlins))
+- 增加跨 Kubernetes 集群滚动重启/缩容 TiCDC 集群的能力 ([#5040](https://github.com/pingcap/tidb-operator/pull/5040), [@charleszheng44](https://github.com/charleszheng44))
```
Comment (ran-huang) [inline zh/releases/release-1.4.5.md:16]:
```suggestion
- 增加跨 Kubernetes 集群滚动重启或缩容 TiCDC 集群的能力 ([#5040](https://github.com/pingcap/tidb-operator/pull/5040), [@charleszheng44](https://github.com/charleszheng44))
```

File: zh/releases/release-1.4.5.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+
+## Bug 修复
```
Comment (ran-huang) [inline zh/releases/release-1.4.5.md:18]:
```suggestion
## 错误修复
```

File: zh/releases/release-1.4.5.md [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+
+- 在新创建的计划备份中取消 GC ([#4940](https://github.com/pingcap/tidb-operator/pull/4940), [@oliviachenairbnb](https://github.com/oliviachenairbnb))
```
Comment (ran-huang) [inline zh/releases/release-1.4.5.md:20]:
```suggestion
- 在新创建的定时备份中取消 GC ([#4940](https://github.com/pingcap/tidb-operator/pull/4940), [@oliviachenairbnb](https://github.com/oliviachenairbnb))
```

File: en/releases/release-1.4.5.md [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+
+Release date: June TODO, 2023
+
+TiDB Operator version: 1.4.5
+
+## Improvements
+
+- Add the metrics for fine-grained TidbCluster reconcile errors ([#4952](https://github.com/pingcap/tidb-operator/pull/4952), [@coderplay](https://github.com/coderplay))
+- Add the metrics for the reconciler and worker queue to improve observability ([#4882](https://github.com/pingcap/tidb-operator/pull/4882), [@hanlins](https://github.com/hanlins))
+- Introduce `startUpScriptVersion` field for DM master to specify startup script version ([#4971](https://github.com/pingcap/tidb-operator/pull/4971), [@hanlins](https://github.com/hanlins))
```
Comment (ran-huang) [inline en/releases/release-1.4.5.md:15]:
```suggestion
- Introduce `startUpScriptVersion` field for DM master to specify the startup script version ([#4971](https://github.com/pingcap/tidb-operator/pull/4971), [@hanlins](https://github.com/hanlins))
```

File: en/releases/release-1.4.5.md [hunk 7]
```diff
@@ -0,0 +1,21 @@
...
+- Add rolling restart and scale in support for TiCDC when deploying TiCDC across multiple kubernetes clusters ([#5040](https://github.com/pingcap/tidb-operator/pull/5040), [@charleszheng44](https://github.com/charleszheng44))
```
Comment (ran-huang) [inline en/releases/release-1.4.5.md:16]:
```suggestion
- Add support for rolling restart and scaling-in of TiCDC clusters when deploying TiCDC across multiple kubernetes clusters ([#5040](https://github.com/pingcap/tidb-operator/pull/5040), [@charleszheng44](https://github.com/charleszheng44))
```

File: en/releases/release-1.4.5.md [hunk 8]
```diff
@@ -0,0 +1,23 @@
...
+- Add the metrics for the reconciler and worker queue to improve observability ([#4882](https://github.com/pingcap/tidb-operator/pull/4882), [@hanlins](https://github.com/hanlins))
+- Introduce `startUpScriptVersion` field for DM master to specify the startup script version ([#4971](https://github.com/pingcap/tidb-operator/pull/4971), [@hanlins](https://github.com/hanlins))
+- Add support for rolling restart and scaling-in of TiCDC clusters when deploying TiCDC across multiple kubernetes clusters ([#5040](https://github.com/pingcap/tidb-operator/pull/5040), [@charleszheng44](https://github.com/charleszheng44))
+
+## Bug fixes
+
+- Supress GC when scheduled backup is newly created ([#4940](https://github.com/pingcap/tidb-operator/pull/4940), [@oliviachenairbnb](https://github.com/oliviachenairbnb))
+- Make `backupTemplate` optional in backup CR ([#4956](https://github.com/pingcap/tidb-operator/pull/4956), [@Ehco1996](https://github.com/Ehco1996))
+- Fix the issue that TiDB Operator panics if no Kubernetes cluster level permission is configured ([#5058](https://github.com/pingcap/tidb-operator/pull/5058), [@liubog2008](https://github.com/liubog2008))
```
Comment (ran-huang) [inline en/releases/release-1.4.5.md:22]:
```suggestion
- Fix the issue that TiDB Operator panics if no Kubernetes cluster-level permission is configured ([#5058](https://github.com/pingcap/tidb-operator/pull/5058), [@liubog2008](https://github.com/liubog2008))
```

File: en/releases/release-1.4.5.md [hunk 9]
```diff
@@ -0,0 +1,23 @@
...
+- Fix the issue that TiDB Operator may panic if `AdditionalVolumeMounts` is set for TidbCluster ([#5058](https://github.com/pingcap/tidb-operator/pull/5058), [@liubog2008](https://github.com/liubog2008))
```
Comment (ran-huang) [inline en/releases/release-1.4.5.md:23]:
```suggestion
- Fix the issue that TiDB Operator might panic if `AdditionalVolumeMounts` is set for TidbCluster ([#5058](https://github.com/pingcap/tidb-operator/pull/5058), [@liubog2008](https://github.com/liubog2008))
```

**Review by ran-huang — COMMENTED** [review]
rest LGTM

**Review by ran-huang — APPROVED** [review]

**Pull request merged**